packages/ui/src/working-groups/hooks/useMyEarnings.ts

```diff
@@ -0,0 +1,40 @@
+import BN from 'bn.js'
+import { startOfMonth, subHours } from 'date-fns'
+
+import { useAccounts } from '@/accounts/hooks/useAccounts'
+import { RewardPaidEventWhereInput } from '@/common/api/queries'
+
+import { RewardPaidEventFieldsFragment, useGetRewardsQuery } from '../queries'
+
+export interface UseMyEarnings {
+  last24hours: BN | null
+  month: BN | null
+}
+
+export function useMyEarnings(): UseMyEarnings {
+  const { allAccounts } = useAccounts()
+
+  const where: RewardPaidEventWhereInput = {
+    rewardAccount_in: allAccounts.map((account) => account.address),
+    createdAt_gte: startOfMonth(Date.now()).toISOString(),
```

Comment:
This will return [Jun 1st](https://date-fns.org/v2.22.1/docs/startOfMonth) for today ;)